examples/python/rippleadd-async.py

```diff
@@ -120,24 +120,27 @@ def unmajority(p, a, b, c):
 qobjs = []
 # Create online (so I/O bound) jobs if we have connetion or local (so CPU bound)
 # jobs otherwise
-if offline == False:
+if offline is False:
     print("Creating %d online jobs..." % NUM_JOBS)
     for _ in range(0, NUM_JOBS):
+
         qobjs.append(qp.compile(["rippleadd"], backend=online_backend,
-                coupling_map=None, shots=1024))
+                                coupling_map=None, shots=1024))
+
+        print("Creating %d local jobs..." % NUM_JOBS)
+        # Create CPU intensive jobs
+        for _ in range(0, NUM_JOBS):
 
-print("Creating %d local jobs..." % NUM_JOBS)
-# Create CPU intensive jobs
-for _ in range(0, NUM_JOBS):
-    qobjs.append(qp.compile(["rippleadd"], backend=local_backend,
-              coupling_map=None, shots=1024))
+            qobjs.append(qp.compile(["rippleadd"], backend=local_backend,
+                                    coupling_map=None, shots=1024))
+        end = False
 
-end = False
-# This function will be called once all jobs have finished.
 def print_results_callback(results, error=None):
+    """This function will be called once all jobs have finished.
+    """
     if error != None:
         print("There was an error executing the circuits!!: Error = {}".format(error))
-        return
```

Comment:
We actually need this `return` - it's a way for terminating the callback early in case of an error, as if it is not present it will continue into the next `for` loop. Could you please reintroduce it?

Reply:
Er, the formatting on this entire script was screwed up and apparently a bunch of things deleted. What I was trying to do was "git checkout master -- examples/python/rippleadd-async.py" but ended up reverting other commits one by one. Sorry, still getting used to the github way of viewing diffs and accessing other repositories.